### PR TITLE
Show previous image tag if no current tag is present

### PR DESCRIPTION
**Pull request by saschagrunert**
Containers storage does now track the image name history, which can be
utilized by CRI-O to display an image name even if the image got
untagged due to pulling a newer image from a remote resource.

### CODE REVIEW

**Comment by saschagrunert** [comment]
@mrunal @haircommander I think we have to mention that in a dedicated way in the release notes. It affects scheduling (in a positive way) as well as the information which images are available on the node.

File: server/image_list.go [hunk 1]
```diff
@@ -36,6 +36,8 @@ func ConvertImage(from *storage.ImageResult) *pb.Image {
 	repoTags := []string{"<none>:<none>"}
 	if len(from.RepoTags) > 0 {
 		repoTags = from.RepoTags
+	} else if from.PreviousName != "" {
+		repoTags = []string{from.PreviousName + ":<none>"}
```
Comment (vrothberg) [inline server/image_list.go]: I didn't test it but could this lead to situations where we return a wrong (outdated) repoTags? (I didn't look at the call sites)

Comment (saschagrunert) [reply server/image_list.go]: Hm, I don't think so because `<none>` will be always treated as "tag does not exist"

**Pull request merged**